api/src/main/java/com/epam/pipeline/acl/notification/SystemNotificationApiService.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017-2019 EPAM Systems, Inc. (https://www.epam.com/)
```

Comment:
This is not directly connected to the pull request but could I ask you to replace the usage of `SystemNotificationApiService.ADMIN_ROLE` with `AclExpressions.ADMIN_ONLY` in the class?